101-application-gateway-rewrite/azuredeploy.parameters.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
remove parameters that have defaultValues, use GEN values for the others (see BP guide)

Reply:
I'll remove the parameters with defaultValues. But i didn't get 'GEN' values.

Reply:
Done